### PR TITLE
Move every tab to modules

### PR DESCRIPTION
**Pull request by chmelevskij**
Move all the tab loading into modules.

Part of #2433

### CODE REVIEW

**Review by McGiverGim — DISMISSED** [review]

**Comment by chmelevskij** [comment]
No bot, go away, I will do this I promise

**Comment by chmelevskij** [comment]
Moved all of the tabs to modules. Will need to verify it works on android and should be OK to merge.

**Comment by haslinghuis** [comment]
Everything is working as far I have tested. One problem I found is:

- enable auto-connect (did not test without)
- go to cli
- type dump and wait for the output to be finished
- select another tab and we get MSP timeouts. After 10 seconds we can reconnect without a problem.

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - Github identifies PR as mergeable -> PASS
 - PR is not in draft -> FAIL
 - Assigned to a milestone -> FAIL
 - cooling off period lapsed (3/06/2021 21:57:37) -> PASS
 - commit count less or equal to three (7 commits) -> FAIL
 - 'Don't merge' label NOT found -> PASS
 - 'Needs coordination with betaflight' label NOT found -> PASS
 - At least one 'RN:' label found -> FAIL
 - 'Tested' label found -> FAIL
 - assigned to an approver (None) -> FAIL
 - approver count at least three (0 approvers) -> FAIL

**Comment by haslinghuis** [comment]
@chmelevskij can you please rebase?

**Review by haslinghuis — DISMISSED** [review]

File: src/js/main.js
Comment (haslinghuis) [inline src/js/main.js:408]:
```suggestion
                        import("./tabs/configuration").then(({ configuration }) =>
```

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> PASS
 - approver count at least three -> FAIL

**Review by blckmn — APPROVED** [review]

**Pull request merged**
